### PR TITLE
updated README

### PR DESCRIPTION
**Pull request by DennoDin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![](./js/assets/logos/blue.png)
+<img src="./js/assets/logos/blue.png" alt="Logo" width=200>
```
Comment (TsubasaK111) [inline README.md]: `yellow.png` plz.
Also, html should not be used here. Resize the logo manually.

File: README.md [hunk 2]
```diff
@@ -37,4 +43,14 @@ Remote JS Debugging (accessible from `cmd+d`) is hugely beneficial.
...
\ No newline at end of file
+`rm -fr ios/build`
+
+## Our Mission
+<!-- Move this somewhere else? -->
+- Enable those who wish to help in a disaster a means to reach those who need it.
+- Provide live tracking and sharing of key locations in a disaster.
+
+## Developers:
+- Tsubasa Kondo
```
Comment (TsubasaK111) [inline README.md]: linkedin plz!
(reference: https://github.com/cc5-team-space-kids/space-explorer/blob/develop/README.md)

**Review by TsubasaK111 — COMMENTED** [review]
Also, we should add a `CONTRIBUTING.md` and a link to it on README.md
It should outline how people can contact us and contribute.
Example: https://github.com/opengovernment/opengovernment/blob/master/CONTRIBUTING.md

File: README.md [hunk 3]
```diff
@@ -46,11 +46,12 @@ Deleting build files may solve your problems.
 `rm -fr ios/build`
 
 ## Our Mission
-<!-- Move this somewhere else? -->
 - Enable those who wish to help in a disaster a means to reach those who need it.
 - Provide live tracking and sharing of key locations in a disaster.
 
+### [How to Contribute](https://github.com/cc5-team-red/zenni/blob/develop/CONTRIBUTE.md)
```
Comment (TsubasaK111) [inline README.md]: ## Like what we do? [Contribute](./CONTRIBUTE.md) with us!
`## Like what we do? [Contribute](./CONTRIBUTE.md) with us!`

**Review by kimikomotoyama — APPROVED** [review]
I'd definitely want to add more to this later, but for now it's good

**Pull request merged**